onesixtyone.c

```diff
@@ -61,11 +62,10 @@ char* snmp_errors[] = {
 struct {
 	int debug;
 	int log;
-  int port;
-  int print_ip;
+    int port;
+    int print_ip;
```

Comment:
there are mixed indentations in this file do you prefer tabs or spaces? If you want I can commit a beautified source.